### PR TITLE
LPS-154835 Mark inputs as required when it applies and remove bottom margin

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/FormInputGeneralPanel.js [hunk 1]
```diff
@@ -327,12 +327,18 @@ function FormInputMappingOptions({
...
+					fields: fieldset.fields
+						.filter(
+							(field) =>
+								ALLOWED_INPUT_TYPES[field.type]?.includes(
+									inputType
+								) && !selectedFields.includes(field.key)
+						)
+						.map((field) =>
+							field.required
+								? {...field, label: `${field.label}*`}
```
Comment (veroglez) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/FormInputGeneralPanel.js:353]: maybe you can simplify this in the following way: 
`{...field, ...(field.required && {label: ${field.label} * } ) }`

what do you think?

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page-structure/components/item-configuration-panels/FormInputGeneralPanel.js:353]: Um, it definitely looks more cool that way! But with that option, you are creating a new object all times, even when the field is not required, right? The other way we only create a new object when it's needed. What do you think? 🤔 I think I'm merging it without changes for now and we can discuss it tomorrow 🥰

**Comment by sandrodw3** [comment]
ci:forward

**Pull request closed without merging**